iris-mpc-cpu/src/next_gen_protocol/ng_worker.rs

```diff
@@ -271,6 +296,44 @@ pub async fn open_t_many(session: &Session, shares: VecShare<u64>) -> eyre::Resu
     Ok(res)
 }
 
+pub async fn open_t_many_16(session: &Session, shares: Vec<Share<u16>>) -> eyre::Result<Vec<u16>> {
```

Comment:
Maybe this function can be deduplicated with open_t_many if you implement From<Vec<Ringelement<T>> and try_into<Vec<Ringelement<T>> for NetworkValue

Reply:
great idea! have done this in here: 8adbcb76755314804a36321128f6e7cc101d63d2